static/openapi/prerendering-spec.json

```diff
@@ -2303,56 +2462,62 @@
-                }
-              }
-            }
-          },
-          "301": {
-            "description": "Moved permanently",
+            "description": "Successful response",
             "content": {
               "application/json": {
                 "schema": {
                   "type": "object",
-                  "description": "Moved permanently",
+                  "description": "Successful response",
                   "properties": {
                     "status": {
                       "type": "string",
-                      "description": "Error status"
+                      "description": "Success status"
                     },
                     "message": {
                       "type": "string",
-                      "description": "Error message"
-                    }
-                  },
-                  "required": ["status", "message"]
-                }
-              }
-            }
-          },
-          "308": {
-            "description": "Permanent redirect",
-            "content": {
-              "application/json": {
-                "schema": {
-                  "type": "object",
-                  "description": "Permanent redirect",
-                  "properties": {
-                    "status": {
-                      "type": "string",
-                      "description": "Error status"
+                      "description": "Success message"
                     },
-                    "message": {
-                      "type": "string",
-                      "description": "Error message"
+                    "data": {
+                      "type": "object",
+                      "properties": {
+                        "_key": {
+                          "type": "string",
+                          "description": "Prefetch configuration unique identifier"
+                        },
+                        "origin": {
+                          "type": "string",
+                          "format": "uri",
+                          "description": "Origin base URL"
+                        },
+                        "cronSchedule": {
+                          "type": "string",
+                          "description": "Cron expression for scheduling"
+                        },
+                        "isRenderForMobile": {
+                          "type": "boolean",
+                          "description": "Whether to render for mobile devices or not"
+                        },
+                        "createdAt": {
+                          "type": "string",
+                          "format": "date-time",
+                          "description": "Date when the prefetch configuration was created"
+                        },
+                        "updatedAt": {
+                          "type": "string",
+                          "format": "date-time",
+                          "description": "Date when the prefetch configuration was updated"
```

Comment:
What format is this date/time format?

Reply:
This is the format: `2023-11-28T12:32:28.108Z`

Reply:
Do you want me to change something here?